.github/workflows/build.yaml

```diff
@@ -18,8 +18,15 @@ jobs:
         with:
           distribution: temurin
           java-version: 17
+      - name: touch local props
+        run: |
+          touch local.properties
```

Comment:
Do we still need to create this file for the root project? I'm asking because it seems like it's only needed for `demo-app`.

Reply:
Ok let's try it without!